app/assets/javascripts/patients.coffee

```diff
@@ -37,6 +54,26 @@ ready = ->
   $(document).on "change", "#abortion-information-form input, #external_pledges input", ->
     updateBalance()
 
+  $(document).on "change", field_selectors, ->
+    i = 0
+    empty = true
+    el = $('#patient_fulfillment_fulfilled')
+
+    while i < fields.length
+      if $(fields[i]).val().length > 0
+        empty = false
+        if el.prop 'checked'
+          break;
+        else
+          el.prop 'checked', true
+        i++
+      else
+        i++
+    if empty == true
+          el.prop 'checked', false
+
+    $(this).submit()
```

Comment:
Not sure we need this, it should be submitting per L31 in `patients.coffee` -- can you confirm?